### ISSUE TITLE
Details List: Group header doesn't announce Expand/Collapse state

### ISSUE THREAD

**Issue by bcoard** [OfficeDev/office-ui-fabric-react#11816]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

When Narrator is turned on and the user uses keyboard navigation, when keyboard focus lands on the group header its expand/collapse state isn't announced.

### Please provide a reproduction of the issue in a codepen:

1. Navigate to [this](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/grouped) link.
2. Turn on Narrator.
3. Using keyboard navigation go to the group header.
